### PR TITLE
MS2.0 MVP

### PR DESCRIPTION
**Pull request by joenpera**
- Configuration cleanup
- Configurations generated in startup
- **CBMA enabled** and CBMA MVP integration

### CODE REVIEW

**Review by TIISR — APPROVED** [review]
No time and can't wait

File: modules/sc-mesh-secure-deployment/src/nats/cbma/macsec/macsec.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        while retries < self.MAX_RETRIES:
+            retries += 1
+            self.__cleanup()
+
+            logger.info(f"Creating MACsec connection with {self.peer_mac} peer")
+            if not run_script_bug_workaround_retcode(create_str.split()):
+                logger.info(f"Monitoring MACsec connection with {self.peer_mac} peer")
+                while run_script_bug_workaround_retcode(monitor_str.split(),
+                                                        quiet_on_error=True) == self.KEY_REFRESH_CODE \
+                      and self.refresh_keys():
```
Comment (pentestiing) [inline modules/sc-mesh-secure-deployment/src/nats/cbma/macsec/macsec.py:111]: Comment this line plz :key:

**Review by pentestiing — CHANGES_REQUESTED** [review]
:crab:

**Review by pentestiing — APPROVED** [review]
:flatbread:

**Pull request merged**